components/Button/button.html

```diff
@@ -0,0 +1,237 @@
+                    towards it.
+                </div>
+
+                <div class="example-content ">
+                    <button class="btn btn-primary"><i class="fa-solid fa-cart-plus"></i> ADD TO CART</button>
+
+                    <button class="btn btn-error"><i class="fa-light fa-heart"></i> ADD TO WISHLIST</button>
+
+                    <button class="btn btn-success"><i class="fa-solid fa-bag-shopping"></i> BUY NOW</button>
+                </div>
+                <iframe src="https://carbon.now.sh/embed/o2atTOPrigPBUqgU0iPO" style="
+              width: 50rem;
+              height: 21rem;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            " sandbox="allow-scripts allow-same-origin">
+                </iframe>
+
+
+
+                <div class="divider-space"></div>
+
+                <div class="content-header">
+                    Float Action Button
+                </div>
+
+                <div class="content-subheading">
+                    These buttons  performs  most common action on screen and are fixed to a particular part of the viewport and are stacked on top of all the
+                    things.
+
+                </div>
+
+                <div class="example-content ">
+                    <button class="float-btn btn-primary"><i class="fa-solid fa-arrow-up-from-line"></i></button>
+
+                    <button class="float-btn btn-success"><i class="fa-solid fa-user-headset"></i></button>
+                </div>
+
+                <iframe src="https://carbon.now.sh/embed/sa7WJDrTuVNN5jBuILFy" style="
+              width: 50rem;
+              height: 21rem;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            " sandbox="allow-scripts allow-same-origin">
+                </iframe>
+
+                <div class="divider-space"></div>
+
+                <div class="content-header">
+                    Button with link
+                </div>
+
+                <div class="content-subheading">
+                   Button with links and basically used when you have to redirect the user onto the another website or on diffrent segment on the same webpage.
+                </div>
+
+                <div class="example-content ">
+                    <button class="btn link-btn"><a href="#">CHROME-link</a></button>
+
+                    <button class="btn link-btn"> <a href="#">EDGE-link</a></button>
+
+                </div>
+
+                <iframe src="https://carbon.now.sh/embed/yiEj5AGguZPXSt9UdM3b" style="
+              width: 50rem;
+              height: 21rem;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            " sandbox="allow-scripts allow-same-origin">
+                </iframe>
+
+            </div>
+        </div>
+    </div>
+    <footer class="footer">
+        <div class="footer-header">Made by Shivangi Sharma</div>
```

Comment:
You can name this class, footer-heading maybe. Just a suggestion

Reply:
ok